### DISCUSSION TITLE
OOM errors

### DISCUSSION THREAD

**Discussion by zohimchandani** [jax-ml/jax#10543] [Q&A]
What does one substitute for `platform` in `XLA_PYTHON_CLIENT_ALLOCATOR=platform`?

**Comment by jakevdp** [comment]
The valid options are `["default", "platform", "bfc", "cuda_async"]`. See https://github.com/tensorflow/tensorflow/blob/77004536aff979d029dccd12880dfa9179933442/tensorflow/compiler/xla/python/xla_client.py#L73

I don't know of any detailed documentation for this setting.
